docs/reference/ui/components.md

```diff
@@ -0,0 +1,182 @@
+(ui_components)=
+
+# UI components
+
+This document provides a reference for Wagtail's user interface (UI) components, which are used to build the features within the CMS. A demonstration of these components are available through the [Styleguide app](styleguide) and the [pattern library](pattern_library).
+
+````{note}
+```{include} ../../../client/README.md
+:start-after: <!-- STABILITY:START -->
+:end-before: <!-- STABILITY:END -->
+```
+````
+
+## Fragment
+
+```{eval-rst}
+.. autofunction:: wagtail.admin.templatetags.wagtailadmin_tags.fragment
+```
```

Comment:
I documented this because it's mentioned in how the "Button with dropdown" component is used

`:no-contents-entry:` isn't supported by `autofunction`, so we still end up with the extra sub item in the TOC.